WooCommerce/Classes/POS/Presentation/ProductCardView.swift

```diff
@@ -14,7 +15,7 @@ struct ProductCardView: View {
             VStack {
                 Text(product.name)
                     .foregroundStyle(Color.primaryBackground)
-                Text(product.priceWithCurrency)
+                Text(product.formattedPrice)
```

Comment:
Agree, this was just a renaming but I think we need to rework the current way currency is applied to the prices. We can also see some duplicity in the entry point, where the dependency is injected directly to the view but is not necessary since this can be located through the view model. I've added a note to tackle this separately.